.gitignore

```diff
@@ -2,4 +2,5 @@ bin/
 
 # resulting materials from running in-toto
 foo.py
-untar.link
\ No newline at end of file
+untar.link
+go.sum
```

Comment:
The Go team's recommendation is to also commit `go.sum` - see [this article](https://github.com/golang/go/wiki/Modules#should-i-commit-my-gosum-file-as-well-as-my-gomod-file).

Is there any reason why we shouldn't commit it here?

Reply:
@radu-matei 
It is premature to add go.su.
We often faced checksum mismatch error. Because when golang version update, it can be occurred by sometimes changing checksum algorithm...
see following.
https://github.com/golang/go/issues/29278

Reply:
Yeah, that makes sense, unfortunately..